package.json

```diff
@@ -27,17 +27,18 @@
         "readable-stream": "4.7.0"
     },
     "devDependencies": {
+        "@metrics/semantic-release-config": "1.0.0",
         "@semantic-release/changelog": "6.0.3",
         "@semantic-release/commit-analyzer": "13.0.1",
         "@semantic-release/git": "10.0.1",
-        "@semantic-release/github": "11.0.6",
-        "@semantic-release/npm": "12.0.2",
+        "@semantic-release/github": "12.0.2",
+        "@semantic-release/npm": "13.1.2",
         "@semantic-release/release-notes-generator": "14.1.0",
```

Comment:
We can remove these since `@metrics/semantic-release-config` manages them for us

Reply:
I assume we remove all `semantic-release*` pakcages then, including `semantic-release` the main package?